### DISCUSSION TITLE
我家毛孩有過敏怎麼辦？用這招讓毛孩擺脫過敏！

### DISCUSSION THREAD

**Discussion by Guagua1234** [1101-GS4719-C/Python-Community-GS4719-C#1141] [General]
https://www.thenewslens.com/feature/petzoo/179217
我覺得飼料中單一動物性蛋白飼料、特殊肉源飼料及水解蛋白飼料的用途及原理。而單一動物性及特殊肉源飼料可用於已知寵物過敏原的情況，而水解蛋白飼料則可用於未知過敏原的情況或進行食物排除試驗。我覺得水解蛋白飼料的原理是將大分子蛋白水解成小分子蛋白，以降低過敏反應。

**Comment by 110302550** [comment]
自己本身也有過敏，如果毛小孩有過敏，完全可以感同身受，這個飼料可以降低過敏反應，對他們來說真的很友善呢
